fairseq/modules/dynamic_convolution.py

```diff
@@ -16,7 +16,7 @@
 def DynamicConv(
     input_size,
     kernel_size=1,
-    padding_l=None,
+    padding_l=1,
```

Comment:
should this be 0?

Reply:
It can be any value greater than or equal to 0. However, it can't be None.